app/views/task/show.html.erb

```diff
@@ -1,23 +1,36 @@
-<h1>Here is the task from <%= "#{@task.name}" %> you requested</h1>
+<h1> View Task </h1>
+<h2>Here is the task from <%= "#{@task.name}" %> you requested</h2>
 
 <table>
   <thead>Task</thead>
+
   <tr>
     <th> ID </th>
     <th> Name </th>
     <th> Description </th>
     <th> Date</th>
-    <th> Created at</th>
-    <th>Updated at </th>
-    <th>Completed </th>
+    <th> Assigned Person </th>
+    <th> Created on</th>
+    <th> Updated at </th>
+    <th> Completed </th>
+    <th> Update </th>
   </tr>
+
   <tr>
     <td>  <%= @task.id %> </td>
     <td>  <%= @task.name %> </td>
     <td>  <%= @task.description %> </td>
-    <td>  <%= @task.date %> </td>
-    <td>  <%= @task.created_at %> </td>
+    <td>
+      <% if @task.date != nil%>
+        <%= @task.date.month %>/<%= @task.date.day %>/<%= @task.date.year %>
```

Comment:
You can also use the `strftime` method to format dates - http://apidock.com/ruby/DateTime/strftime